src/mapeo-manager.js

```diff
@@ -78,23 +80,34 @@ export class MapeoManager extends TypedEmitter {
    * @param {Object} opts
    * @param {Buffer} opts.rootKey 16-bytes of random data that uniquely identify the device, used to derive a 32-byte master key, which is used to derive all the keypairs used for Mapeo
    * @param {string} opts.dbFolder Folder for sqlite Dbs. Folder must exist. Use ':memory:' to store everything in-memory
+   * @param {string} [opts.projectMigrationFolder] path for drizzle migration folder for project
+   * @param {string} [opts.clientMigrationFolder] path for drizzle migration folder for client
```

Comment:
Think we should be explicit about these options and not have them be optional with a default value

```suggestion
   * @param {string} opts.projectMigrationsFolder path for drizzle migrations folder for project database
   * @param {string} opts.clientMigrationsFolder path for drizzle migrations folder for client database
```